### PR TITLE
Extracted mongo DB service and S3 service 

### PR DESCRIPTION
**Pull request by tamas-molnar**

### CODE REVIEW

**Comment by tosan88** [comment]
well, lgtm

File: MongoService.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
+package main
```
Comment (catalin-ilea) [inline MongoService.go]: Name conventions for files is lower case: services.go maybe?

File: S3WriterProvider.go [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package main
```
Comment (catalin-ilea) [inline S3WriterProvider.go]: s3_provider.go ?

Comment (Nesze) [inline S3WriterProvider.go]: better: s3-provider.go

**Pull request merged**